plugin/go.vim

```diff
@@ -26,14 +26,14 @@ function! s:checkVersion() abort
     endif
 
     if l:unsupported == 1
-      echohl Error
-      echom "vim-go requires Vim 7.4.2009 or Neovim 0.3.2, but you're using an older version."
-      echom "Please update your Vim for the best vim-go experience."
-      echom "If you really want to continue you can set this to make the error go away:"
-      echom "    let g:go_version_warning = 0"
-      echom "Note that some features may error out or behave incorrectly."
-      echom "Please do not report bugs unless you're using Vim 7.4.2009 or newer or Neovim 0.3.2."
-      echohl None
+      call go#util#EchoError([
```

Comment:
This used echom on purpose since older Vim version may error out on autoloading the `autoload/go/util.vim` file and/or executing the `EchoError` function.

Reply:
Again, I changed this deliberately. If it errors trying to load `autoload/go/util.vim`, then things are messed up pretty badly anyway likely the `go#config#VersionWarning` already failed.

Reply:
Probably a good reason to not use `go#config#VersionWarning()` here, too.

I didn't do this for the fun of it, but in response to real errors. The entire reason this exists is to present a nice "your Vim is too old" message instead of confusing parse errors and the like.

Reply:
yeah, fair. I'll revert this bit.